src/components/Form/Form.tsx

```diff
@@ -11,7 +11,7 @@ const Form = ({ updateUserSentiment }: FormProps): JSX.Element => {
 
   const submitUserSentiment = (userSentiment: number) => {
       updateUserSentiment(userSentiment);
-      history.push("/feed");
+      history.push("/feed/");
```

Comment:
See similar comment above
```suggestion
      history.push("/feed");
```